snapshot/src/main/java/io/camunda/zeebe/snapshots/impl/SnapshotMetrics.java

```diff
@@ -78,4 +86,8 @@ void observeSnapshotFileSize(final long sizeInBytes) {
   Timer startTimer() {
     return SNAPSHOT_DURATION.labels(partitionId).startTimer();
   }
+
+  Timer startPersistTimer() {
+    return SNAPSHOT_PERSIST_DURATION.labels(partitionId).startTimer();
```

Comment:
:x: Since the label is not changing once the instance is created, can you use child timers to avoid recreating it every time? See `MetaStoreMetrics` for example. Ole found that it had some impact to call `#labels` every time.

Reply:
Yeah I also thought also about it but I didn't wanted to change the whole class, I wanted to keep the change small to be honest 😅 Since we should do it everywhere.

If I find time I can do it.